lib/connector-handler.ts

```diff
@@ -49,8 +49,30 @@ export interface Context {
 	[prop: string]: any
 
 	reloadConfig(): Promise<any>
+	assumeAwsRole(externalRoleInput: ExternalRoleInput): Promise<CredentialResponse>;
+}
+export class ExternalRoleInput {
+	externalArnRole: string;
```

Comment:
Could we call this `roleArn` to line up with AssumeRole API call we'll eventually make?